Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2125,7 +2147,7 @@ class CVE(Indicator):
         :return: None
         :rtype: ``None``
         """
-        CONTEXT_PATH = 'URL(val.Data && val.Data == obj.Data)'
+        CONTEXT_PATH = 'CVE(val.ID && val.ID == obj.ID)'
```

Comment:
why don't we have dbotscore for CVE ?

Reply:
CVEs has no DBotScores. It is not an indicator like IP.
There was an old discussion whether we want CVEs have DBot Score and at the end decided not to have.
@michalgold correct me if I am wrong